src/Features/Core/Portable/Completion/Providers/Snippets/AbstractSnippetCompletionProvider.cs

```diff
@@ -88,11 +88,12 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
             foreach (var snippetData in snippets)
             {
                 var completionItem = SnippetCompletionItem.Create(
-                    displayText: snippetData.DisplayName,
+                    displayText: snippetData.SnippetIdentifier + " snippet",
```

Comment:
temporarily there until we do not remove conflicting named items from completion list